### PR TITLE
chore(readme): redesign

### PR DESCRIPTION
**Pull request by moshe-blox**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,59 +1,60 @@
...
+- [SSV API Docs](https://ssvlabs.github.io/ssv/)
+
+## <img src="https://github.com/Anmol-Baranwal/Cool-GIFs-For-GitHub/assets/74038190/fa83eeb9-f4e2-4d85-93f0-688af11babf8" width="35">&nbsp; Contribution
 
 Thank you for considering a contribution to the source code.
 
-In order to contribute to eth2-ssv, please fork, add your code, commit and send a pull request
+In order to contribute to `ssv`, please fork, add your code, commit and send a pull request
 for the maintainers to review and merge into the main code base.\
 If you wish to submit more complex changes though, please check up with the core devs first on [our discord](https://discord.gg/eDXSP9R)
```
Comment (iurii-ssv) [inline README.md:59]: Is this Discord link working for you guys ? 

image

It should probably match the one we have below: https://discord.gg/ssvnetworkofficial ?

Comment (nkryuchkov) [reply README.md:59]: same here, let's change it while we're touching this
image

File: README.md [hunk 2]
```diff
@@ -1,59 +1,60 @@
...
-* [iBFT annotated paper (By Blox)](./ibft/IBFT.md)
-* [EIP650](https://github.com/ethereum/EIPs/issues/650)
-* [Security proof for n-t honest parties](https://notes.ethereum.org/DYU-NrRBTxS3X0fu_MidnA)
-* [MEV Research - Block proposer/ builder separation in SSV](https://hackmd.io/DHt98PC_S_60NbnW4Wgssg)
+- Article by [SSV](https://medium.com/bloxstaking/an-introduction-to-secret-shared-validators-ssv-for-ethereum-2-0-faf49efcabee)
+
+### 🔷 Technical iBFT and SSV read
+
+- [iBFT Paper](https://arxiv.org/pdf/2002.03613.pdf)
+- [iBFT annotated paper (By Blox)](./ibft/IBFT.md)
```
[the remark this answers is not in the document]
Comment (moshe-blox) [reply README.md]: fixed

File: README.md (hunk 1 above)
Comment (moshe-blox) [reply README.md]: thx, fixed

**Review by y0sher — APPROVED** [review]

**Review by iurii-ssv — APPROVED** [review]

**Review by nkryuchkov — APPROVED** [review]

**Pull request merged**